### DISCUSSION TITLE
flag meaning [Oct 26]

### DISCUSSION THREAD

**Discussion by dstala** [nocodb/nocodb#4269] [Discord Threads]
> 🙋‍♂️ **KVY**  
## Hello guys! What does it mean? (in column properties)  
![image0](https://cdn.discordapp.com/attachments/661905455894888493/1032205571249668106/unknown.png)

**Comment by o1lab** [comment]
> 🙋‍♂️ **wkw**  
#### not null, primary key, auto increment, unsigned, auto update timestamp (e.g. updated_at)  
#  
> 🙋‍♂️ **Vanderbijl**  
#### I thought un was unique value?  
#  

 [Autogenerated from discord](https://discord.com/channels/661905455894888490/1032205571589419028)  
- - - -  
#### Join [NocoDB's](http://www.nocodb.com/)  community  
[Discord](https://discord.gg/5RgZmkW) • [Twitter](https://twitter.com/nocodb) • [Reddit](https://www.reddit.com/r/NocoDB/) • [Docs](https://docs.nocodb.com/)
